src/reducers/items.js

```diff
@@ -167,6 +173,14 @@ export default function reducer(state = initialState, action) {
         items: state.items.filter(item => item.id !== id),
       };
     }
+    case CHANGE_LIST_NAME_REQUEST: {
+      return { ...state, loading: true, error: null };
+    }
+    case CHANGE_LIST_NAME_SUCCESS: {
+      return { ...state, name: action.name, editingName: !state.editingName };
```

Comment:
You should reset loading to false and actually set editing to false rather than flipping it.